bioblend/_tests/TestGalaxyWorkflows.py

```diff
@@ -113,16 +114,35 @@ def test_import_export_workflow_from_local_path(self):
             shutil.rmtree(export_dir)
         self.assertIsInstance(exported_wf_dict, dict)
 
+    def test_import_publish_export_workflow_from_local_path(self):
+        with self.assertRaises(Exception):
+            self.gi.workflows.import_workflow_from_local_path(None)
+        path = test_util.get_abspath(os.path.join('data', 'paste_columns.ga'))
+        imported_wf = self.gi.workflows.import_workflow_from_local_path(path, publish=True)
+        self.assertIsInstance(imported_wf, dict)
+        self.assertFalse(imported_wf['deleted'])
+        self.assertTrue(imported_wf['published'])
+
     def test_import_export_workflow_dict(self):
         path = test_util.get_abspath(os.path.join('data', 'paste_columns.ga'))
         with open(path, 'r') as f:
             wf_dict = json.load(f)
         imported_wf = self.gi.workflows.import_workflow_dict(wf_dict)
         self.assertIsInstance(imported_wf, dict)
         self.assertFalse(imported_wf['deleted'])
+        self.assertFalse(imported_wf['published'])
         exported_wf_dict = self.gi.workflows.export_workflow_dict(imported_wf['id'])
         self.assertIsInstance(exported_wf_dict, dict)
 
+    def test_import_publish_export_workflow_dict(self):
```

Comment:
Can you remove `export_` from the method name?

Reply:
Fixed by https://github.com/galaxyproject/bioblend/pull/247/commits/7e2a16b2bff1d56d44311c88bf761f273e94ddb0